hoot-core/src/main/cpp/hoot/core/elements/Tags.cpp

```diff
@@ -235,6 +236,29 @@ QStringList Tags::getNames() const
   return result;
 }
 
+QStringList Tags::getNamesRecursive() const
+{
```

Comment:
Made that change and it looks good with the tests....pushed to 5722 branch.  Do I need to create a new pull request now?  Or is it possible to make the original one pick up that change?